### ISSUE TITLE
Expected issues with Site portion of upcoming releases

### ISSUE THREAD

**Issue by hazendaz** [mybatis/mybatis-dynamic-sql#723]
@jeffgbutler Due to some issues you noted long back regarding our then current site strategy to release sites, I made efforts to use maven directly.  Unfortunately this is a complete and utter mess.  So I don't want to impose upon you any releases taking hours to release the site (mybatis core took 1 hour 22 minutes just on the site).  So when you have a release you need to get out, I'll take the burn on it and do it for you until this can be figured out.  I'm basically always available.  I'm hoping maven team will assist as its a complete mess.  I did want to make sure you were very aware as I don't see you over on slack that often and just wanted to make sure you are aware while this project will release fine, it will take forever and if any multi module builds those will just crash out and force some manual way which is not working at all with the parent structure we use.  There are work arounds that require a lot of manual stuff but its really bad to impose that on anyone that isn't expecting it.

**Comment by jeffgbutler** [comment]
@hazendaz - thanks for the heads up and I appreciate your work on this. 

I wrote some notes about doing a site release manually here: https://github.com/mybatis/mybatis-dynamic-sql/blob/master/RELEASING.md

Honestly, it's not that difficult to do it this way and only takes a few minutes.

**Comment by hazendaz** [comment]
Hi @jeffgbutler read over that, seems extra complicated but while working would only work at the moment with single pom build (non multi module).  

I have put in place an action that isn't across all of mybatis yet but would allow you to just git push what you want to a branch called 'site' and it do it for you.

The problem is still multi module.  I'll be looking into that but your current way would in fact avoid the long delay I mention.  If it does however seem to go so out of control on timing, don't waste your time, waste mine.  I feel like its my fault from trying to go native maven when clearly I've yet to see a project use direct maven for these exact reasons.  Honestly I don't get why they have it so complicated when it should in theory be super easy.  

And to be honest, they are hard to work with.  I raised issues like many others on their modular approach to javadocs for example and they did nothing for 4 full years before it finally affected them and they fixed it exactly like told to do so 4 years earlier.  So far my complaints or asks on the site stuff has gone no where :(  It doesn't help they document most of their stuff in change logs and never on real documentation far too often.

At any rate, just was really wanting to make sure you were aware we took this direction after discussion on slack and its not worked out like anticipated.  For simple repos, it works fine.  Its just more complex ones that are problematic :(
